release_bot/cli.py

```diff
@@ -31,6 +31,9 @@ def parse_arguments():
                             default='')
         parser.add_argument("-v", "--version", help="display program version", action='version',
                             version=f"%(prog)s {configuration.version}")
+        parser.add_argument("-n", "--dryrun", default = False, 
```

Comment:
```suggestion
        parser.add_argument("-n", "--dry-run", default = False, 
```

See for example [git add](https://git-scm.com/docs/git-add#_options)

Reply:
I also prefer separator in case of variables (e.g `dry_run`) WDYT?

Reply:
`dry-run` would not be a valid variable name.

On the other hand, `dry_run` seems to be perfect.